src/snapred/backend/dao/request/ReductionRequest.py

```diff
@@ -0,0 +1,16 @@
+from typing import List, Literal, Union
+
+from pydantic import BaseModel
+
+from snapred.backend.dao.state.FocusGroup import FocusGroup
+
+
+class ReductionRequest(BaseModel):
+    runNumber: str
+    useLitemode: bool
+    focusGroup: List[FocusGroup]
+    calibrantSamplePath: str
+
+    smoothingParameter: float = Config["calibration.parameters.default.smoothing"]
```

Comment:
This should get pulled from a normalization record

Reply:
Agreed, I will edit SousChef to pull it directly